src/redux/features/OTPSlice.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { OTPState } from '../../@types/auth/loginTypes';
+import { DynamicData } from '../../@types/DynamicData';
+import API from '../../utils/api';
+
+export const verifyOTP = createAsyncThunk(
+	'verifyOTP',
+	async (
+		{ otp, token }: { otp: string; token: string },
+		{ rejectWithValue },
```

Comment:
this restructured paramaters should be in interface instead of directly pass them as destrctured props.

```
interface Props{
otp: string; 
token: string 
}
```

then we have  `{ otp, token }: Props`